src/core/middlewares/interceptor/quota/quota.go

```diff
@@ -135,7 +135,8 @@ func (qi *quotaInterceptor) computeResources(req *http.Request) error {
 		return nil
 	}
 
-	if len(qi.opts.Resources) == 0 && qi.opts.OnResources != nil {
+	qi.resources = qi.opts.Resources
```

Comment:
what's the difference?

Reply:
delete chart is using opt.Resources(types.ResourceList{types.ResourceCount: 1}, but it still cannot satisfy the condition?

Reply:
quotaInterceptor reserve and unreserve will process the resources saved in quotaInterceptor struct.

This change ensure that resources will first get from the Resources of opts, if it is empty, then try to compute resources.

Reply:
ok，got it.